tests/test_other.py

```diff
@@ -2331,7 +2331,7 @@ def test_llvm_nativizer(self):
     output = Popen([os.path.join(self.get_dir(), 'files.o.run')], stdin=open(os.path.join(self.get_dir(), 'stdin')), stdout=PIPE, stderr=PIPE).communicate()
     self.assertContained('''size: 37
 data: 119,97,107,97,32,119,97,107,97,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35,35
-loop: 119 97 107 97 32 119 97 107 97 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 35 
```

Comment:
hmm, that whitespace actually matters. i pushed a fix to incoming now so it doesn't look unnecessary to editors.

Reply:
Rebased against incoming